components/bounty/funding-section/fund-modal.tsx

```diff
@@ -11,18 +11,22 @@ import Modal from "components/modal";
 
 import { ApplicationContext } from "contexts/application";
 import { useIssue } from "contexts/issue";
+import { useNetwork } from "contexts/network";
 import { toastError, toastSuccess } from "contexts/reducers/add-toast";
 
 import { formatNumberToCurrency } from "helpers/formatNumber";
 
+import useApi from "x-hooks/use-api";
 import useBepro from "x-hooks/use-bepro";
 import useERC20 from "x-hooks/use-erc20";
 
 import { Amount, CaptionMedium, RowWithTwoColumns } from "./minimals";
 
 export default function FundModal({
   show = false,
-  onCloseClick
+  onCloseClick,
+  repoId,
+  ghId
```

Comment:
We are using the `issue` context in this component, so we don't need to pass theses ids by props